src/Bonsai.ML.Torch/NeuralNets/Models/ModelArchitecture.cs

```diff
@@ -0,0 +1,23 @@
+namespace Bonsai.ML.Torch.NeuralNets.Models
+{
+    /// <summary>
+    /// Represents the architecture of a neural network model.
+    /// </summary>
+    public enum ModelArchitecture
+    {
+        /// <summary>
+        /// The AlexNet model architecture.
+        /// </summary>
+        AlexNet,
+
+        /// <summary>
+        /// The MobileNet model architecture.
+        /// </summary>
+        MobileNet,
+
+        /// <summary>
+        /// The MNIST model architecture.
+        /// </summary>
+        MNIST
```

Comment:
```suggestion
        Mnist
```

Pascal casing to follow acronym naming conventions.